### PR TITLE
FLUT-6853 - [Feature] Added the UG documentation content for the completed features

### PR DESCRIPTION
**Pull request by Tamilarasan-Paranthaman**
### Description ###

- Added the content for the `unsort/sortable` icon feature.
- Resaved all the images in the sorting topic, since we show the sortable icon in the column header.
- Added the cell focus enhancement in the editing `canCellSubmit` topic.
- Added the` images` in the DataGrid `Editing` topic(support improvement).
- Added the details in the selection cell style topic (support improvement).

### CODE REVIEW

**Comment by Tamilarasan-Paranthaman** [comment]
rebuild

**Comment by Tamilarasan-Paranthaman** [comment]
rebuild

File: Flutter/datagrid/selection.md [hunk 1]
```diff
@@ -1613,5 +1613,5 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized current cell](images/selection/flutter-datagrid-customized-currentcell.png)
 
-
-
+>**NOTE**
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/selection.md:1616]: You can change the current row border's color by the `currentCellStyle` property of `SfDataGridThemeData` in `SfDataGridTheme`. The current row border is shown when `navigationMode` is `row` to navigate between rows.

File: Flutter/datagrid/sorting.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ documentation: ug
...
+The datagrid provides the built-in support to sort one or more columns by setting the [SfDataGrid.allowSorting](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/allowSorting.html) property to true. When sorting is applied, the datagrid automatically rearranges the data to match with the current sort criteria. When `SfDataGrid.allowSorting` is true, you can sort the data simply by tapping the column header. 
+
+By default, the datagrid shows unsort icon on every column header to indicating the sorting is enabled in the respective column. After tapping on column header dtagrid shows a respective sort icon in the column header to indicating the sort direction.
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/sorting.md:14]: By default, the datagrid shows unsort icon on every column header to indicate the sorting is enabled in the respective column. When sorting is applied to columns, datagrid shows a respective sort icon in the column header to indicate the sort direction.

**Review by NeelakandanKannan — CHANGES_REQUESTED** [review]
I have added my feedback as comments. Get back to me with the changes.

File: Flutter/datagrid/selection.md (hunk 1 above)
Comment (Tamilarasan-Paranthaman) [reply Flutter/datagrid/selection.md:1616]: Modified

File: Flutter/datagrid/sorting.md (hunk 2 above)
Comment (Tamilarasan-Paranthaman) [reply Flutter/datagrid/sorting.md:14]: Modified

**Review by NeelakandanKannan — APPROVED** [review]
The chnages are fine.

**Pull request merged**